### ISSUE TITLE
Remove duplicate "Full-resolution download"

### ISSUE THREAD

**Issue by rexagod** [publiclab/Leaflet.DistortableImage#151]
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/PublicLab.Editor/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update:** Remove any one of the duplicate "Full-resolution download" sections from the [README](https://github.com/publiclab/Leaflet.DistortableImage/blob/main/README.md) file.
```markdown
## Full-resolution download

We've added a GPU-accelerated means to generate a full resolution version of the distorted image; it requires two additional dependencies to enable; see how we've included them in the demo:

<script src="../node_modules/webgl-distort/dist/webgl-distort.js"></script>
<script src="../node_modules/glfx/glfx.js"></script>
****
```
 
- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

**Comment by mridulguptaa** [comment]
i would like to contribute here. This will be my first contribution. Please assign it to me

**Comment by gauravano** [comment]
Go ahead @mridulguptaa!

**Comment by mridulguptaa** [comment]
Pull requeset created  @gauravano 
#152

**Comment by Aneal-Sharma** [comment]
I would like to contribute!

**Comment by JARIS101** [comment]
I would Love to Contribute!

**Comment by dkstm95** [comment]
I would like to contribute! Please assign it to me!

**Comment by jywarren** [comment]
Hi, all, this was solved in #152 - thank you @mridulguptaa ! For others, we can post some new first-timers issues soon - please keep an eye on https://code.publiclab.org/#r=all - and THANK YOU!!!
